### DISCUSSION TITLE
Why ChatGPT plus account is required？

### DISCUSSION THREAD

**Discussion by jackywu** [acheong08/ChatGPT-Proxy-V4#4] [Q&A]
thanks for your work, and , can you tell me why ChatGPT plus account is required other than normal account？

**Comment by acheong08** [comment]
`_puid` only shows up on plus accounts.

**Reply by jackywu** [reply]
Thanks, by the way, i want to setup myself proxy like https://bypass.duti.tech/api/conversation without chatgpt Plus,  is this viable? Is the proxy code open source?
